src/main/java/com/crud/tasks/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -27,7 +27,11 @@
     @Scheduled(fixedDelay = 10000)
     public void sendInformationEmail(){
         long size = taskRepository.count();
-        simpleEmailService.send(new Mail(adminConfig.getAdminMail(),SUBJECT,"Currently in database you got: "+ size + " tasks"));
+        String message = "Currently in database you got: "+ size + " tasks";
+        if(size == 1){
+            message = "Currently in database you got: 1 task";
+        }
```

Comment:
to rozwiazanie lamie zasade DRY
Zapisz np.
String suffix = size > 1 ? "task":"tasks";
`String.format("Currently in database you got: %s %s, size, suffix")`